script/start-gke-env.sh

```diff
@@ -29,10 +29,14 @@ fi
 
 echo "Creating cluster $CLUSTER in $ZONE (v$VERSION)"
 gcloud container clusters create --cluster-version=$VERSION --zone $ZONE $CLUSTER --num-nodes 5 --machine-type=n1-standard-2
-# Wait for the cluster to respond
+echo "Waiting for the cluster to respond..."
 cnt=20
```

Comment:
So this means that this 20 seconds check limit was not working right? So now, is it 20 seconds enough?  What's in your experience the average time that takes a cluster to be reachable?

Maybe we should play it safer? At the end of the day for example 60 seconds does not seem a big deal to me

Reply:
Yes this didn't work because as far as I can tell `((cnt=cnt-1))` will never fail, it will just keep subtracting into negative integers.

I agree that we might want to try 60 here though.

Reply:
these are actually retries not exactly seconds (it depends in how long `kubectl get pods` takes). In my experience if after a couple of retries this doesn't work it's because there is an issue and it won't work. Note that `gcloud container clusters create` only returns control when the cluster is healthy and `.kube/config` is properly configured unless there is an error.

Let's keep this short, if we find errors with this loop after getting a success in the cluster creation we can increase it, in other case we will just be adding overhead for failed builds.

Reply:
Thanks for the explanation!

So then the concept of time here is not accurate then no? Maybe it should be called retries?

Reply:
agreed, we should rename this to retries.